### ISSUE TITLE
con you download only new chapters ?

### ISSUE THREAD

**Issue by imed-ghomari** [manga-py/manga-py#400]
# Feature request

**Is your feature request related to a problem? Please describe.**
Instead of having only the option of downloading all the chapters of the manga

**Describe the solution you'd like**
I would like to follow a manga and download only the new chapters, -> to have the option to specify the last read chapter so that it only downloads the chapters after this one 

Is this feature already available or not ? if not please add something of the sort, thank you.

**Comment by 1271** [comment]
You can.

Please skip chapters (see `--skip-volumes` / `-s`)

For example:

```
manga-py http://example.com/awesome/manga -s 20
```

Skips the first 20 chapters.


---


You can also specify the number of chapters to download (`--max-volumes` / `-c`):


For example:

```
manga-py http://example.com/awesome/manga -c 5
```

Download the first 5 chapters.

---

You can also load chapters in reverse order (`--reverse-downloading` / `-r`).

```
manga-py http://example.com/awesome/manga -r
```

---

Perhaps you mean the ability to specify the chapter as indicated on the site?
(I mean just the text. It will be difficult for some sites) 

Not sure if this is a good idea.

**Comment by imed-ghomari** [comment]
Using these options, the closest you can get to my request is downloading **multiple manga** with **reverse order** with a **maximum chapter of 1 or 2**, but even then it will re-download chapters that have already been downloaded. Hence, the options that I suggested : 
- to have a list of manga in your favorites and
- remember the last downloaded chapter for each one
- then with one command, the option to check each manga on the list for a new, not yet downloaded chapter and download only these.

Other nice options to have : 
- to remember the last downloaded chapter for each manga even if the files have been deleted
- An easy way to add **favorites** and **last chapter read** manually to the json file

Is this possible ?

**Comment by 1271** [comment]
Not now.
You can either implement something like this for yourself, or just not delete the chapters. (this is the default behavior) 

In theory, something like this could be implemented. 🤔 


Although all manga-py will do is keep the index of the last chapter for each url
It's real, yeah

**Comment by imeed166** [comment]
Yeah, you're right, I tried and it didn't re-download the chapters. However, I can't download multiple manga at once it seems.

**Comment by imed-ghomari** [comment]
Yeah, you're right, I tried and it didn't re-download the chapters.

**Comment by 1271** [comment]
The point is that I don't want to add support for loading some chapters now (batch)
Some part of the manga-py will just break because of this xD
But I can add a flag that saves the chapter index and restores it the next run

**Comment by imed-ghomari** [comment]
Yes, saving the chapter index without keeping the file would be nice

**Comment by 1271** [comment]
Ok, I will add the parameter a little later

**Comment by imed-ghomari** [comment]
Thank you, and the ability to only download chapters that have an index greater than the one saved would be awesome (sorry last request xp)

**Comment by imed-ghomari** [comment]
like this maybe :
`manga-py http://example.com/awesome/manga -s -index
`

**Comment by 1271** [comment]
Please, check new version `pip install "manga-py>=1.31.0"`
